.gitignore

```diff
@@ -4,3 +4,4 @@ build
 package-lock.json
 target
 Cargo.lock
+./build
```

Comment:
remove the ., and please use newlines at the end of files

Reply:
That's another typo. Should be the `.build/` folder.